scene/gui/tree.cpp

```diff
@@ -1258,6 +1258,13 @@ void TreeItem::deselect(int p_column) {
 	_cell_deselected(p_column);
 }
 
+void TreeItem::clear_buttons() {
+	for (int i = 0; i < cells.size(); ++i) {
+		cells.write[i].buttons.clear();
+		cells.write[i].cached_minimum_size_dirty = true;
+	}
+}
```

Comment:
This should either call `_changed_notify();` or have a comment why it's not doing so. The only current use do more modifications after it, so it won't make any difference, but if it is called on its own, it won't update column size and queue redraw.